packages/core/src/utils/bigNumberToLocaleString.ts

```diff
@@ -0,0 +1,38 @@
+import BigNumber from 'bignumber.js';
+
+const complexNumber = 1234567.0123456789;
+
+export default function bigNumberToLocaleString(value: BigNumber, locale?: string): string {
+    const formatter = Intl.NumberFormat(locale);
+    if (!formatter) {
+      throw new Error(`Formater for ${locale} is not supported`);
```

Comment:
```suggestion
      throw new Error(`Formatter for ${locale} is not supported`);
```

Reply:
We should add try-catch into a function that is using this function. It is correct to throw an Error but it is not correct if we are not going to catch it. I will check if I am using try-catch for this. The second option is to use always "default" locale which will return the correct value

Reply:
@paninaro I already updated typo.